examples/issue/components/IssueEditor.js

```diff
@@ -36,7 +36,7 @@ export default class IssueEditor extends React.Component {
     };
   }
 
-  onChange = (editorState, cb) => this.setState({ editorState }, cb);
+  onChange = (editorState) => this.setState({ editorState });
```

Comment:
I see places in `draft-js-plugins-editor` where an object is being passed as the second argument to `onChange`.